cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/SavedAttachment.java

```diff
@@ -57,7 +57,7 @@
     private final File file;
     private final AttachmentStreamFactory attachmentStreamFactory;
 
-    protected SavedAttachment(long seq, String name, byte[] key, String type, Encoding encoding,
+    public SavedAttachment(long seq, String name, byte[] key, String type, Encoding encoding,
```

Comment:
This is another awkward one I hope we might be able to reconsider this when we revisit the `AttachmentManager`.